src/frontend/src/lib/styles/global/button.scss

```diff
@@ -144,4 +144,39 @@ button {
 		background: rgba(0, 0, 0, 0.3);
 		border: 1px solid rgba(255, 255, 255, 0.3);
 	}
+
+	&.authenticate {
+		position: relative;
+		z-index: 0;
+	}
+
+	&.authenticate::before {
+		border-radius: 9999px;
```

Comment:
`border-radius: 50%`?

Reply:
I tried that, but it is not working... I think it is because there is no width, so it becomes like this

<img width="347" alt="Screenshot 2024-08-23 at 11 33 06" src="https://github.com/user-attachments/assets/ea17f440-6356-487b-b5fc-e215f5a17459">

Reply:
Gotcha